Automattic/newspack-blocks
# Hotfix/thank you screen translations

### All Submissions:


### Changes proposed in this Pull Request:

This PR updates the POT file for the Newspack Blocks plugin and regenerates the JSON files. 

See: p1701452156856909-slack-newspack-support

### How to test the changes in this Pull Request:

1. Apply the PR.
2. Switch your site's language to one that's bundled in the plugin (like German).
3. Confirm that languages are loading on the front-end and in the editor -- not all blocks have translations, but trying the Donate Block or Homepage Posts block will show obviously translated strings.
4. Confirm that strings from the thank-you.php template have been added to the POT file (you can search for this file name in the POT file). 

### Other information:
